src/lib.rs

```diff
@@ -16,17 +16,21 @@ This example reads an SVG file and writes the corresponding PDF back to the disk
 # fn main() -> Result<(), Box<dyn std::error::Error>> {
 use svg2pdf::usvg::fontdb;
 use svg2pdf::{ConversionOptions, PageOptions};
+use std::sync::Arc;
 
 let input = "tests/svg/custom/integration/matplotlib/stairs.svg";
 let output = "target/stairs.pdf";
 
 let svg = std::fs::read_to_string(input)?;
-let options = svg2pdf::usvg::Options::default();
 let mut db = fontdb::Database::new();
 db.load_system_fonts();
-let tree = svg2pdf::usvg::Tree::from_str(&svg, &options, &db)?;
+let options = svg2pdf::usvg::Options {
```

Comment:
could also take the `options.fontdb_mut().load_system_fonts()` route. it's a little simpler and doesn't create two databases.